modules/user-guide/partials/benefits-of-PR-review-in-Che.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: PROCEDURE
```

Comment:
I'd say the first choice I'd have to make when setting out to write a module about a topic is to decide on the type of module: PROCEDURE, CONCEPT, or REFERENCE.
https://redhat-documentation.github.io/modular-docs/#_creating_modules
In this particular case, I'd try to figure out whether this is better written as a PROCEDURE module or as a CONCEPT module. The CONCEPT is descriptive writing offers more freedom (and can even have subsections!). The PROCEDURE is basically a strict list of imperative sentences that the intended-audience reader can easily perform to reach a specific outcome.